lib/smart_answer/helpers/format_capture_helper.rb

```diff
@@ -0,0 +1,64 @@
+module SmartAnswer
+  module Helpers::FormatCaptureHelper
+    DEFAULT_FORMATS = {
+      govspeak: [/^body$/, /^post_body$/, /^next_steps$/],
+      text: [/^title$/, /^meta_description$/, /^hint$/, /^label$/, /^suffix_label$/, /^error_*./],
+    }.freeze
+
+    def content_for(name, content = nil, options = {}, &block)
+      if block_given?
+        options = content if content
+        content = capture(&block)
+
+        format = options.fetch(:format, default_format(name))
```

Comment:
Minor: we ought to delete this key from the options, before we pass to the parent.

Reply:
Add this and a test